src/test/java/racingCar/CarTest.java

```diff
@@ -0,0 +1,44 @@
+package racingCar;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import racingCar.model.RacingCarGame;
+import racingCar.model.Car;
+import racingCar.model.RandomMoveStrategy;
+
+import java.util.List;
+
+import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
+import static org.assertj.core.api.AssertionsForClassTypes.assertThatExceptionOfType;
+
+public class CarTest {
+
+    RacingCarGame racingCarGame;
+    private final static int TRY_TIMES = 5;
+    private final static String[] CAR_NAMES = "pobi,crong,honux,min".split(",");
+
+    @BeforeEach
+    void beforeTest() {
+        racingCarGame = new RacingCarGame(CAR_NAMES, TRY_TIMES);
+    }
+
+    @Test
+    void 자동차_리스트_생성_테스트() {
+        List<Car> cars = racingCarGame.getRaceCar();
+        assertThat(cars.size()).isEqualTo(4);
+    }
+
+    @Test
+    void 자동차_이름_5글자_초과_테스트() {
+        assertThatExceptionOfType(IllegalArgumentException.class)
+                .isThrownBy(() -> {
+                    Car car = new Car("minchoi", new RandomMoveStrategy());
+                });
+    }
+
+    @Test
+    void 자동차_동작_테스트() {
+        Car car = new Car("pobi",  new RandomMoveStrategy());
+        // 어떤 식으로 True / False를 나눠야 할지 잘 모르겠음.
+    }
```

Comment:
```suggestion
    @Test
    void 자동차_동작_테스트() {
        Car car = new Car("pobi", () -> true);
        // 어떤 식으로 True / False를 나눠야 할지 잘 모르겠음.
        car.move();
        int moveCount = car.getMoveCount();
        assertThat(moveCount).isOne();
    }

```
현재 코드기반으로 짠다면 이렇게 고쳐볼 수 있을 것 같아요!
자동차가 움직이지 않을 때의 테스트 코드를 짠다면 `assertThat(moveCount).isZero()` 로 바꾸면 되구요:)
위의 코드가 이해가 안되는 부분이 있다면, `함수형 인터페이스` 에 대해 공부해 보시면 좋을 것 같습니다!

위에 피드백에 따라 move에 전략패턴을 사용해서 고치는것이 좋을 것 같아요 :)
Car객체에서 이동전략까지 가지고 있을 필요는 없다고 생각해요, 이동 전략은 언제든지 바뀔 수 있는 부분인데 만약 전략이 바뀌면 setStrategy 하고 move를 또 호출해야 되는 상황이 생기게 되죠, move에서 전략만 받으면 되는일인데 말이죠..! 🤔 
때문에 MoveStrategy는 move()라는 메서드의 파라미터로 외부에서 주입받아 동작하는 방식으로 설계하는 것이 더 객체지향적인 코드인 것 같아요.